traffic_ops/traffic_ops_golang/deliveryservice/keys.go

```diff
@@ -127,16 +127,27 @@ func AddSSLKeys(w http.ResponseWriter, r *http.Request) {
 
 	api.CreateChangeLogRawTx(api.ApiChange, "DS: "+*req.DeliveryService+", ID: "+strconv.Itoa(dsID)+", ACTION: Added/Updated SSL keys", inf.User, inf.Tx.Tx)
 
+	var alerts tc.Alerts
+
 	if isUnknownAuth {
-		api.WriteRespAlert(w, r, tc.WarnLevel, "WARNING: SSL keys were successfully added for '"+*req.DeliveryService+"', but the input certificate may be invalid (certificate is signed by an unknown authority)")
-		return
+		alerts.AddAlert(tc.Alert{
+			Text:  "WARNING: SSL keys were successfully added for '" + *req.DeliveryService + "', but the input certificate may be invalid (certificate is signed by an unknown authority)",
```

Comment:
can we take out the "WARNING: " and "Successfully " while we're at it? That information is given to you by the Level. Imagine logging these prefixed by their level:
> WARNING: WARNING: SSL keys were successfully added for 'something', but the input certificate may be invalid (certificate is signed by an unknown authority)
> SUCCESS: Successfully added ssl keys for something